src/components/admin/settings-tab.yml

```diff
@@ -13,15 +13,24 @@ footerLinksHeader:
 defaultTagsHeader:
   en: Default tags
   de: Vordefinierte Tags
-createDocumentRegenerationBatchHeader:
+createDocumentRegenerationRequestHeader:
   en: Document recreation
```

Comment:
I'd go for `regeneration` here, cause `recreation` I think is rather used in the "leasure" sense. Or 're-creation'.

Reply:
Yes, agree, done